src/o365/yammer/commands/user/user-list.ts

```diff
@@ -0,0 +1,195 @@
+import commands from '../../commands';
+import GlobalOptions from '../../../../GlobalOptions';
+import {
+  CommandOption, CommandValidate
+} from '../../../../Command';
+import YammerCommand from "../../../base/YammerCommand";
+import request from '../../../../request';
+
+const vorpal: Vorpal = require('../../../../vorpal-init');
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  letter?: string;
+  sortBy?: string;
+  reverse?: boolean;
+  limit?: number;
+  groupId?: number;
+}
+
+class YammerUserListCommand extends YammerCommand {
+  protected items: any[];
+
+  /* istanbul ignore next */
+  constructor() {
+    super();
+    this.items = [];
+  }
+
+  public get name(): string {
+    return `${commands.YAMMER_USER_LIST}`;
+  }
+
+  public get description(): string {
+    return 'Returns users from the current network';
+  }
+
+  public getTelemetryProperties(args: CommandArgs): any {
+    const telemetryProps: any = super.getTelemetryProperties(args);
+    telemetryProps.letter = args.options.letter !== undefined;
+    telemetryProps.sortBy = args.options.sortBy !== undefined;
+    telemetryProps.reverse = args.options.reverse !== undefined;
+    telemetryProps.limit = args.options.limit !== undefined;
+    telemetryProps.groupId = args.options.groupId !== undefined;
+    return telemetryProps;
+  }
+
+  private getAllItems(cmd: CommandInstance, args: CommandArgs, page: number): Promise<void> {
+    return new Promise<void>((resolve: () => void, reject: (error: any) => void): void => {
+      if (page === 1) 
```

Comment:
Let's wrap this in `{ }` to avoid formatting issues.

Reply:
@waldekmastykarz: sorry, I don't get here what you exactly want. Would you please show me what you mean exactly?

Reply:
To use `{` and `}` for opening and closing blocks. So instead of 
```
if(statement)
    block of code
```
to turn into 
```
if(statement) {
    block of code 
}
```